src/odb/include/odb/dbObject.h

```diff
@@ -90,6 +90,7 @@ enum dbObjectType
   dbPolygonObj,
   dbPowerDomainObj,
   dbPowerSwitchObj,
+  dbPropertyObj,
```

Comment:
Reordering the values here will change their numeric value.  Usages like _dbAccessPoint::vias_ will need backward compatibility code for a schema rev.

Reply:
I am picking up on a few other classes like dbMarker. I can't think of an easy way to handle this to be backward compatible.